auto_round/utils.py

```diff
@@ -559,12 +595,13 @@ def get_layer_names_in_block(model, supported_types=[torch.nn.Linear, transforme
         if isinstance(m, tuple(supported_types)):
             m.tmp_name = n
     layers_in_block = []
-    block_names = get_block_names(model)
-    for block_name in block_names:
-        block = get_module(model, block_name)
-        for n, m in block.named_modules():
-            if hasattr(m, "tmp_name"):
-                layers_in_block.append(m.tmp_name)
+    all_blocks = get_block_names(model, multimodal)
```

Comment:
is there any way to check whether it is a mutlimodal model from model itself/inputs?

Reply:
For a multimodal model there should be a arg that determines whether or not to quantize the part of vision, and the arg here are for that purpose, not to identify the model type.